### ISSUE TITLE
Feature Request: Button to take a picture

### ISSUE THREAD

**Issue by phmarek** [markusfisch/BinaryEye#148]
Related to my use case in #147, here's another feature request: Please provide a button (VolDown, or an on-screen one) to take a picture of the currently selected sub-frame and POST that to a second configurable URL.

(I just tried to walk through one shelf and found out that quite a subset doesn't have a barcode with the ISBN; if I could just press some button to instead take a picture of the title and author, or a text representation of th ISBN, capturing the required data would be much easier; of course, the post-processing is still necessary, but with some quick OCR library on the server even that would be less painful than the alternatives.

Yeah, sorry, BinaryEye becomes "just another sort of camera app", then ;/

**Comment by markusfisch** [comment]
Well, I see why this would be convenient but wouldn't taking a screenshot (by pressing Power + Volume Down) accomplish _almost_ the same? 😬 Of course, the screenshots would end up in `/sdcard/Pictures/Screenshots` (or just in "Screenshots" in your gallery app), but it wouldn't be hard to take them from there 🤔

**Comment by phmarek** [comment]
> but wouldn't taking a screenshot (by pressing Power + Volume Down) accomplish _almost_ the same?

No. The screenshot keys depend on the Android version AFAIK, and have quite some delay.

On my phone they're on different sides, so I'd need to switch the way I'm holding my mobile, most probably losing the direction to point.


Getting the right moment for a screenshot (text in focus and visible) is much harder than just moving the thumb a bit.

**Comment by markusfisch** [comment]
Hm, yes, I see. Still, this is a very special use case 🤔

**Comment by phmarek** [comment]
> Still, this is a very special use case 🤔 

Well, it's just good design - provide an easy fallback solution in case the main function fails in some way.
